flake8-docstrings is not needed to run tests.
See also https://github.com/python-versioneer/python-versioneer/issues/398

--- versioneer-0.29/tox.ini.orig
+++ versioneer-0.29/tox.ini
@@ -27,7 +27,6 @@
 [testenv]
 deps =
     flake8
-    flake8-docstrings
     wheel>=0.35
     setuptools>=50
     virtualenv>=20
